### PR TITLE
Changed param to hash map

### PR DESCRIPTION
**Pull request by itzikPichApplicaster**
changed interface from kotlin to java

### CODE REVIEW

File: src/main/java/com/applicaster/playerevents/PlayerEventsProvider.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.applicaster.playerevents;
+
+import java.util.HashMap;
+
+public interface PlayerEventsProvider {
+
+    void onPlayerEvents(String event, HashMap<String,Object> properties);
```
Comment (pablorueda) [inline src/main/java/com/applicaster/playerevents/PlayerEventsProvider.java]: Can you name it onPlayerEvents in singular, without the s, as we are receiving just one event.

File: src/main/java/com/applicaster/playerevents/PlayerEventsManager.kt [hunk 2]
```diff
@@ -16,8 +16,8 @@ object PlayerEventsManager {
         }
     }
 
-    fun onPlayerEvent(event: String, properties: Any) {
-        playerEventsProviders.forEach { it.onPlayerEvent(event,properties) }
+    fun onPlayerEvent(event: String, properties: HashMap<String,Any>) {
+        playerEventsProviders.forEach { it.onPlayerEvents(event,properties) }
```
Comment (pablorueda) [inline src/main/java/com/applicaster/playerevents/PlayerEventsManager.kt]: Can you name it onPlayerEvents in singular, without the s, as we are receiving just one event.

**Review by pablorueda — CHANGES_REQUESTED** [review]
Asked to rename the method from plural to singular.

File: src/main/java/com/applicaster/playerevents/PlayerEventsProvider.java (hunk 1 above)
Comment (itzikPichApplicaster) [reply src/main/java/com/applicaster/playerevents/PlayerEventsProvider.java]: will be fixed

File: src/main/java/com/applicaster/playerevents/PlayerEventsManager.kt (hunk 2 above)
Comment (itzikPichApplicaster) [reply src/main/java/com/applicaster/playerevents/PlayerEventsManager.kt]: will be fixed

**Pull request merged**